doc/adr/0008-load-multiple-rdata-files.md

```diff
@@ -0,0 +1,18 @@
+# 8. As a researcher you are able to load multiple RData files
```

Comment:
Is that within one study? Or all studies the researcher has access too?

Reply:
If you want rights on a specific file in a folder it is not possible. Maybe I should write that down in here?

Reply:
If a researcher participates in multiple studies, he can load data files from all studies. (But the tables are not explicitly namespaced, if different .RData files have different versions of the same table only the last one loaded will survive.)